data/campaigns/The_Rise_Of_Wesnoth/scenarios/17a_The_Dragon.cfg

```diff
@@ -25,21 +25,24 @@
         unrenamable=yes
         side=1
         canrecruit=yes
-        {GOLD 230 200 170}
+        {GOLD 290 230 170}
+        {INCOME 6 4 2}
         controller=human
         team_name=Haldric
         user_team_name=_"Refugees"
         {FLAG_VARIANT loyalist}
     [/side]
 
+    {STARTING_VILLAGES 1 {ON_DIFFICULTY 7 5 3}}
+
     [side]
         type=Saurian Flanker
         id=Irix
         name= _ "Irix"
         side=2
         canrecruit=yes
         {GOLD 80 100 120}
-        {INCOME 3 6 9}
+        {INCOME 1 5 9}
```

Comment:
Income varies too much here I think. Please keep the original.

Reply:
ok, reverted in 3f99f17